src/nvim/buffer_defs.h

```diff
@@ -649,7 +649,7 @@ struct file_buffer {
   bool b_p_swf;                 /* 'swapfile' */
   long b_p_smc;                 /* 'synmaxcol' */
   char_u      *b_p_syn;         /* 'syntax' */
-  long b_p_ts;                  /* 'tabstop' */
+  int b_p_ts;                   // 'tabstop'
```

Comment:
I will revert this change and cast to dissociate this PR from the "numerical options" problem so this PR can be merged independently.